Source/AliveLibAO/BaseAliveGameObject.cpp

```diff
@@ -166,10 +166,15 @@ void BaseAliveGameObject::VOnPathTransition(__int16 camWorldX, int camWorldY, Ca
 
 __int16 BaseAliveGameObject::VTakeDamage(BaseGameObject* /*pFrom*/)
 {
-    // Empty
+    // Defaults to no damage.
     return 0;
 }
 
+__int16 BaseAliveGameObject::VTakeDamage_401920(BaseGameObject* pFrom)
+{
+    return VTakeDamage(pFrom);
```

Comment:
you've done this back to front, this needs to be empty/return 0 and VTakeDamage should be calling VTakeDamage_401920